bin/sbang

```diff
@@ -109,13 +109,17 @@ while read line && ((lines < 2)) ; do
     fi
     lines=$((lines+1))
 done < "$script"
+#this is ineeded for scripts with sbang parameter
+#like ones in intltool
```

Comment:
Can you add spaces after your comments? I wish there were an equivalent of flake8 for bash...

Also, an example would be nice. Something like:
```bash
# Some shebangs add flags at the end:
#     #!/path/to/bin/perl -w
# We need to hold on to these flags
```
Also, `interpreter_v` and `interpreter_f` aren't very clear variable names...